docs/docs/query-execution.md

```diff
@@ -113,17 +113,17 @@ We now have the list of all pages that need to be executed (linked to their Quer
 }
 ```
 
-This Query Job contains everything we need to execute the query (and do things like recording dependencies between pages and nodes). So, we push it onto the queue in [query-queue.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/internal-plugins/query-runner/query-queue.js) and then wait for the queue to empty. Let's see how `query-queue` works.
+This Query Job contains everything you need to execute the query (and do things like recording dependencies between pages and nodes). So, you push it onto the queue in [query-queue.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/internal-plugins/query-runner/query-queue.js) and then wait for the queue to empty. Let's see how `query-queue` works.
 
 #### Query Queue Execution
 
-[query-queue.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/internal-plugins/query-runner/query-queue.js) creates a [better-queue](https://www.npmjs.com/package/better-queue) queue that offers advanced features like parallel execution, which is handy since queries do not depend on each other so we can take advantage of this. Every time an item is consumed from the queue, we call [query-runner.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/internal-plugins/query-runner/query-runner.js) where we finally actually execute the query!
+[query-queue.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/internal-plugins/query-runner/query-queue.js) creates a [better-queue](https://www.npmjs.com/package/better-queue) queue that offers advanced features like parallel execution, which is handy since queries do not depend on each other so you can take advantage of this. Every time an item is consumed from the queue, you call [query-runner.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/internal-plugins/query-runner/query-runner.js) where you finally actually execute the query!
 
 Query execution involves calling the [graphql-js](https://graphql.org/graphql-js/) library with 3 pieces of information:
 
 1. The Gatsby schema that was inferred during [Schema Generation](/docs/schema-generation/).
 1. The raw query text. Obtained from the Query Job.
-1. The Context, also from the Query Job. Has the page's `path` amongst other things so that we can record [Page -> Node Dependencies](/docs/page-node-dependencies/).
+1. The Context, also from the Query Job. Has the page's `path` amongst other things so that you can record [Page -> Node Dependencies](/docs/page-node-dependencies/).
```

Comment:
```suggestion
1. The Context, also from the Query Job. Has the page's `path` amongst other things so that Gatsby can record [Page -> Node Dependencies](/docs/page-node-dependencies/).
```